pkg/agent/openflow/pod_connectivity_test.go

```diff
@@ -0,0 +1,362 @@
+			enableIPv4:       true,
+			skipWindows:      true,
+			trafficEncapMode: config.TrafficEncapModeNetworkPolicyOnly,
+			expectedFlows: []string{
+				"cookie=0x20000000000, table=ARPSpoofGuard, priority=200,arp,in_port=2,arp_spa=10.10.0.1,arp_sha=00:00:10:10:00:01 actions=goto_table:ARPResponder",
+				"cookie=0x20000000000, table=ARPResponder, priority=200,arp,arp_op=1 actions=move:NXM_OF_ETH_SRC[]->NXM_OF_ETH_DST[],set_field:aa:bb:cc:dd:ee:ff->eth_src,set_field:2->arp_op,move:NXM_NX_ARP_SHA[]->NXM_NX_ARP_THA[],set_field:aa:bb:cc:dd:ee:ff->arp_sha,move:NXM_OF_ARP_TPA[]->NXM_NX_REG2[],move:NXM_OF_ARP_SPA[]->NXM_OF_ARP_TPA[],move:NXM_NX_REG2[]->NXM_OF_ARP_SPA[],INPORT",
+				"cookie=0x20000000000, table=ARPResponder, priority=190,arp actions=NORMAL",
+				"cookie=0x20000000000, table=Classifier, priority=200,in_port=2 actions=set_field:0x2/0xf->reg0,goto_table:SpoofGuard",
+				"cookie=0x20000000000, table=SpoofGuard, priority=200,ip,in_port=2 actions=goto_table:UnSNAT",
+				"cookie=0x20000000000, table=ConntrackZone, priority=200,ip actions=ct(table=ConntrackState,zone=65520,exec(nat))",
+				"cookie=0x20000000000, table=ConntrackState, priority=210,ct_state=+inv+trk,ip actions=drop",
+				"cookie=0x20000000000, table=ConntrackState, priority=190,ct_state=-new+trk,ip actions=goto_table:AntreaPolicyEgressRule",
+				"cookie=0x20000000000, table=L3Forwarding, priority=210,ip,nw_dst=10.10.0.1 actions=set_field:00:00:10:10:00:01->eth_dst,set_field:0x20/0xf0->reg0,goto_table:L3DecTTL",
+				"cookie=0x20000000000, table=L3Forwarding, priority=210,ct_state=+rpl+trk,ct_mark=0x2/0xf,ip actions=set_field:00:00:10:10:00:01->eth_dst,set_field:0x20/0xf0->reg0,goto_table:L3DecTTL",
+				"cookie=0x20000000000, table=L3Forwarding, priority=190,ip actions=set_field:00:00:10:10:00:01->eth_dst,set_field:0x20/0xf0->reg0,goto_table:L3DecTTL",
+				"cookie=0x20000000000, table=L3Forwarding, priority=0 actions=set_field:0x20/0xf0->reg0,goto_table:L2ForwardingCalc",
+				"cookie=0x20000000000, table=L3DecTTL, priority=210,ip,reg0=0x2/0xf actions=goto_table:SNATMark",
+				"cookie=0x20000000000, table=L3DecTTL, priority=200,ip actions=dec_ttl,goto_table:SNATMark",
+				"cookie=0x20000000000, table=L2ForwardingCalc, priority=200,dl_dst=00:00:10:10:00:01 actions=set_field:0x2->reg1,set_field:0x100/0x100->reg0,goto_table:IngressSecurityClassifier",
+				"cookie=0x20000000000, table=IngressSecurityClassifier, priority=210,pkt_mark=0x80000000/0x80000000,ct_state=-rpl+trk,ip actions=goto_table:ConntrackCommit",
+				"cookie=0x20000000000, table=ConntrackCommit, priority=200,ct_state=+new+trk-snat,ct_mark=0x0/0x10,ip actions=ct(commit,table=Output,zone=65520,exec(move:NXM_NX_REG0[0..3]->NXM_NX_CT_MARK[0..3]))",
+				"cookie=0x20000000000, table=ConntrackState, priority=0 actions=goto_table:PreRoutingClassifier",
+				"cookie=0x20000000000, table=Output, priority=200,reg0=0x100/0x100 actions=output:NXM_NX_REG1[]",
+			},
+		},
+		{
```

Comment:
Windows does not support PolicyOnly mode

Reply:
I'll remove that.